docker-compose.yml

```diff
@@ -5,15 +5,20 @@ services:
     image: acouvreur/ssh-log-to-influx
     build: ./
     restart: always
-    environment: 
+    environment:
       - INFLUX_URL=influxdb
       - INFLUX_DB=telegraf
+      - NODE_TLS_REJECT_UNAUTHORIZED=0
+      - INFLUX_PROTOCOL=https
+      - INFLUX_USER=$INFLUX_USER
+      - INFLUX_PWD=$INFLUX_PWD
```

Comment:
`- INFLUX_PWD=$INFLUX_PWD` 
is the same as 
`- INFLUX_PWD`
it will pass the current var to the container